### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by chrisamiller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,339 +1,31 @@
...
+# Analysis Workflows for Genomic Analysis
 
-Our main concern so far has been getting CWLs converted to
-WDL. Following this will be efforts on optimization of the workflows,
-and cleanup of the repository.
+A collection of WDL workflows for analysis of genomic sequencing data. These enable primary analysis of many common experiments, including somatic and germline variant calling, RNA sequencing, epigenomic assays, and ommunogenomics approaches for cancer vaccine development.
 
-In the future we may rework the structure of this repository to
-a format that Dockstore supports and leverage that tool.
+These pipelines were developed by the [Washington University Division of Oncology](https://oncology.wustl.edu/), in collaboration with the [McDonnell Genome Institute](https://genome.wustl.edu/) and other members of the [Washington University School of Medicine](https://medicine.wustl.edu/).  Much of the code and structure is adapted from a sister repository of [workflows in the CWL language](https://github.com/genome/analysis-workflows/)
```
Comment (tmooney) [inline README.md:5]:
```suggestion
These pipelines were developed by the [Washington University Division of Oncology](https://oncology.wustl.edu/), in collaboration with the [McDonnell Genome Institute](https://genome.wustl.edu/) and other members of the [Washington University School of Medicine](https://medicine.wustl.edu/).  Much of the code and structure is adapted from a sister repository of [workflows in the CWL language](https://github.com/genome/analysis-workflows/).
```

File: README.md [hunk 2]
```diff
@@ -1,339 +1,31 @@
...
-If the failed write happened at `/cromwell_root` path, then `disks:
-"local-disk ..."` needs to be increased. However, if the failed write
-happens during to `stdin` or `stdout`, or any of the other standard
-Linux-y places, then you'll need to increase the value of
-`bootDiskSizeGb`. Cromwell in GCP mounts two disks, at minimum: the
-boot disk, and a local-disk. Boot disk handles all the operating
-system files, but local-disk is where almost all of your "work" is
-going to happen, besides piping between commands.
+### Workflow Execution
+These are primarily designed to be run with the cromwell workflow system, either on a self-hosted server through the Google Cloud Platform ([details here](https://github.com/wustl-oncology/cloud-workflows)), through [Terra](https://terra.bio/)) or on local compute clusters. They have been extensively used in these contexts, though the portability of WDL means that running on other platforms should be straightforward as well.
```
Comment (tmooney) [inline README.md:16]:
```suggestion
These are primarily designed to be run with the cromwell workflow system, either on a self-hosted server through the Google Cloud Platform ([details here](https://github.com/wustl-oncology/cloud-workflows)), through [Terra](https://terra.bio/)), or on local compute clusters. They have been extensively used in these contexts, though the portability of WDL means that running on other platforms should be straightforward as well.
```

File: README.md [hunk 3]
```diff
@@ -1,339 +1,31 @@
...
-many runs failed because a file wasn't included in the
-instance. Generally, this happens because the CWL did not specify a
-secondaryFile that it assumed would exist next to the passed in
-file. This works on the cluster, because the tools just look for the
-file and it already sits where it's expected. This does not work on
-the cloud because that file is never sent to the instance. The
-solution is to add this parameter explicitly to the WDL and pass it
-through, top down.
+### Data
+Input "bundles" containing reference genomes and annotations are available for use/download along with documentation on how they were created.
```
Comment (tmooney) [inline README.md:22]: This one really cries out for a link 🙂

File: README.md [hunk 4]
```diff
@@ -1,339 +1,31 @@
...
+We answer questions through github issues on this repository, and also have compiled a list of [common errors](https://github.com/wustl-oncology/analysis-wdls/blob/main/docs/common_errors.md) that may be useful.
 
-This is one of two things. Either (A) the input is malformed or
-otherwise incorrect, or (B) the specified file was not uploaded to the
-bucket. These are both instances of the general version of the error,
-"No file has been uploaded to the specified URL".
+## Contributions
 
-# Differences from CWL
+A big thanks to all of the developers, bioinformaticians and scientists who built this resource. For a complete list of software contributions, i.e. commits, to this repository, please see the GitHub Contributors both to [this repository](https://github.com/wustl-oncology/analysis-wdls/graphs/contributors) as well as to the [analysis-workflows](https://github.com/genome/analysis-workflows/graphs/contributors) repo.
```
Comment (tmooney) [inline README.md:29]:
```suggestion
A big thanks to all of the developers, bioinformaticians, and scientists who built this resource. For a complete list of software contributions, i.e. commits, to this repository, please see the GitHub Contributors both to [this repository](https://github.com/wustl-oncology/analysis-wdls/graphs/contributors) as well as to the [analysis-workflows](https://github.com/genome/analysis-workflows/graphs/contributors) repo.
```

File: README.md [hunk 5]
```diff
@@ -1,339 +1,31 @@
...
-a format that Dockstore supports and leverage that tool.
+These pipelines were developed by the [Washington University Division of Oncology](https://oncology.wustl.edu/), in collaboration with the [McDonnell Genome Institute](https://genome.wustl.edu/) and other members of the [Washington University School of Medicine](https://medicine.wustl.edu/).  Much of the code and structure is adapted from a sister repository of [workflows in the CWL language](https://github.com/genome/analysis-workflows/)
 
 
-# Common Errors
+## Getting Started
 
-## Out of Space
+### Workflows
+Download our repository with `git clone https://github.com/wustl-oncology/analysis-wdls.git`
```
Comment (tmooney) [inline README.md:11]:
```suggestion
Download our repository with `git clone https://github.com/wustl-oncology/analysis-wdls.git`.
```

File: README.md [hunk 6]
```diff
@@ -1,339 +1,31 @@
...
 
-These errors indicate that the disk storage space has been
-filled. That part is pretty straightforward. The part that's a bit
-more of a pitfall is that _depending on what output_ this happened on,
-you need to change different disk sizes.
+Terra hosts an excellent guide to [getting started with WDL](https://support.terra.bio/hc/en-us/articles/360037117492-Overview-Getting-started-with-WDL)
```
Comment (tmooney) [inline README.md:13]:
```suggestion
Terra hosts an excellent guide to [getting started with WDL](https://support.terra.bio/hc/en-us/articles/360037117492-Overview-Getting-started-with-WDL).
```

**Review by tmooney — COMMENTED** [review]
Generally looks good to me!  There was at least one serial comma in this document, so I took the liberty of suggesting it be added in a few places it was missing 🙂

File: README.md (hunk 3 above)
Comment (chrisamiller) [reply README.md:22]: Yeah, it's going to get one - on my to-do list for the next couple of days!

**Pull request merged**